OnlineLibrary.Common/Filters/OLExceptionFilter.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using OnlineLibrary.Common.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OnlineLibrary.Common.Filters
+{
+    public class OLExceptionFilter : IExceptionFilter
+    {
+        public void OnException(ExceptionContext context)
+        {
+            Exception exc = context.Exception;
+            var olException = exc as OLException;
```

Comment:
Let's rename to GenericExceptionFilter and leave OLException handling as is but add also default exception handling